jbrowse/src/org/labkey/jbrowse/JBrowseController.java

```diff
@@ -962,14 +962,18 @@ public ApiResponse execute(LuceneQueryForm form, BindException errors)
 
             try
             {
-                return new ApiSimpleResponse(searcher.doSearchJSON(
+                HttpServletResponse response = getViewContext().getResponse();
+                response.setContentType("application/x-ndjson");
+                searcher.doSearchJSON(
                         getUser(),
                         PageFlowUtil.decode(form.getSearchString()),
                         form.getPageSize(),
                         form.getOffset(),
                         form.getSortField(),
-                        form.getSortReverse()
-                ));
+                        form.getSortReverse(),
+                        response
+                );
+                return null;
```

Comment:
do we really want to return null here?

Reply:
should searcher.doSearchJSON() return something instead?

Reply:
My understanding is that for both query endpoints (CSV and JSON) we probably don't want to return a value using the APIResponse returned from execute() because we are manually writing output to the HttpServletResponse in the body of the methods